electron/main.ts

```diff
@@ -646,8 +646,7 @@ ipcMain.handle('egsSync', async (event, args) => {
   }
 })
 
-ipcMain.on('addShortcut', async(event, args) => {
```

Comment:
are you sure that this works?

Reply:
Absolutely, it was duplicated. I think you did when you merged the code in main.

Reply:
it is still there as you can see:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/commit/a7297af8d15bac3d9b4c1c6defe20b374d06f882